docs/references/storage/README.md

```diff
@@ -0,0 +1,281 @@
+
+## Q1 goals & summary of results
+The expected result for Q1 was fixing the problem of storage growth despite pruning and improving database access times with a more optimal key layout. 
+
+While we did indeed fix the problem of pruning/compaction not working, based on the results we obtained we could not demonstrate a certain benefit of changing the database key representation. 
+
+Without a real world application, when running our test applications, our hypothesis on the impact of ordering was demonstrated by performance improvements and faster compaction times with the new layout.
+
+The one experiment with a real application (injective) did not back up this theory though. Even though the overall performance was better than the software version used by the application at the moment. 
+
+The block processing time reported by our application were in the range of 2-6s compared to 100s of ms for the real world application. Furthermore, the application might have different access patterns. Our tests can be seen as testing only CometBFT's interaction with storage, without much interference from the application.  
+
+That is why, in Q1, we introduce an interface with two implementations: the current key layout (a "v1") and a new representation ("v2") sorting the keys by height using ordercode. The new layout is marked as purely experimental. Our hope is that chains will be incentivized to experiment with it and provide us with more real world data. This will also facilitate switching the data layout without breaking changes between releases if we decide to officially support a new data layout. 
+
+**Summary of Q1 results:**
+
+- pruning on its own is inefficient to control storage growth
+- we have complemented pruning with a (forced) compaction feature, and this proved effective to control storage growth
+- we confirmed that moving pruning + compaction to background mitigates potential performance impact of this feature, by running tests with it on Injective mainnet
+  - this works as expected, therefore pruning + compaction does not show to impact node performance and we recommend using it
+- regarding key layouts:
+  - our hypothesis was that the new "v2" layout should improve read/write performance on block & state store access
+  - we obtained contradicting results on this hypothesis locally vs. production settings, concretely:
+  - in local setting, enabling pruning with the old "v1" key layout _had_ a performance impact; enabling pruning with new layout _had no_ impact
+  - in mainnet setting the observation was in reverse: using the new layout _introduced a ~10ms latency_
+  - it is inconclusive if the new key layout "v2" is beneficial, so we will introduce this as an experimental feature
+  - we expect to continue working with operator teams to gather data from production, ideally Injective and Osmosis
+- pebbleDB: handles compaction without the need for Comet to force it, generally shows better performance with the new layout
```

Comment:
Let's add here what exists (pruning, forced compaction, background pruning+compaction) in which versions (v1/main/0.38/0.38-experimental) of CometBFT pls.